src/esthri/src/compression.rs

```diff
@@ -34,6 +35,15 @@ mod bio {
     pub(super) use tempfile::NamedTempFile;
 }
 
+pub(crate) fn compressed_file_metadata() -> HashMap<String, String> {
+    let mut m = HashMap::new();
```

Comment:
I wanted to use `once_cell` here, because this HashMap is literally the same every time, but couldn't manage to get it to work, mainly because the rustoto structs (`CreateMultipartUploadRequest` and `PutObjectRequest`) only accepting values for these metadata HashMaps despite seemingly only ever using the HashMap as a reference anyway. :man_shrugging: Maybe there's a nicer way to go about it that I'm missing?

Reply:
You could use once_cell, but clone the value, but that I think that's pretty much equivalent to this.  Using an iterator to initialize the HashMap would get this on one line, e.g.:

```rust
    HashMap::<_, _>::from_iter(IntoIter::new([("esthri_compress_version".into(), env!("CARGO_PKG_VERSION").into())]))
```

Adapted from https://stackoverflow.com/questions/27582739/how-do-i-create-a-hashmap-literal

Reply:
SO answer also points to this crate: https://docs.rs/maplit/latest/maplit/

Reply:
Ah nice! My current naive approach looks like it might be a little more understandable/readable, so I might just stick with it for now.

Reply:
👍